ocaml/tests/test_extauth_plugin_ADwinbind.ml

```diff
@@ -41,6 +41,50 @@ module ExtractOuConfig = Generic.MakeStateless (struct
       ]
 end)
 
+module Range = Generic.MakeStateless (struct
+  module Io = struct
+    type input_t = int * int * int
```

Comment:
I assume the names are dictated by the functor and can't be changed. I find them a bit verbose.

Reply:
I think so, lots of other test test use it, so let's just keep to for consistency.